### PR TITLE
Add user menu as a dropdown

### LINKED ISSUE #3888: Make the "My name" button in to a dropdown menu
Making in to a dropdown menu allows us to have a place for things that are not specific submission or project.

By moving "Log out" in to this menu we also free up some space.

Start with these items:

- [ ] My account
- [ ] Activity feed (used to be in the notification bell, only for staff)
- [ ] My flagged (only for staff)
- [ ] Wagtail admin (only for staff)
- [ ] Log out

Might be a good idea to change the url for the activity feed from "/activity/notifications/" in to something better. They are not really notifications.

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #3888 

Login button
image
on hover:
image
Staff logged in:
image
on hover:
image
On clicking on Staff button:
image
On hovering any item of these like 'Activity Feed' in this case:
image
Applicant or any other user logged in:
image

## Test Steps

 - [ ] …

### CODE REVIEW

File: hypha/templates/includes/login_button.html [hunk 1]
```diff
@@ -2,23 +2,68 @@
...
-        <a
-            class="button button--transparent button--narrow {{ class }}"
-            href="{{ APPLY_SITE.root_url }}{% url 'dashboard:dashboard' %}"
-        >
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-
-            {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
```
Comment (frjo) [inline hypha/templates/includes/login_button.html]: Add a "button" class here. The adds the small runded corners we have on all buttons now.

File: hypha/templates/includes/login_button.html [hunk 2]
```diff
@@ -2,23 +2,68 @@
...
-        <a
-            class="button button--transparent button--narrow {{ class }}"
-            href="{{ APPLY_SITE.root_url }}{% url 'users:account' %}"
-            title="Goto your account"
-        >
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-            {{ request.user }}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
```
Comment (frjo) [inline hypha/templates/includes/login_button.html]: Add a "button" class here.

File: hypha/templates/includes/login_button.html [hunk 3]
```diff
@@ -2,23 +2,68 @@
...
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-
-            {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
+                {% heroicon_micro "user-circle" class="inline align-text-bottom w-8 h-8 me-1" aria_hidden=true %}
+                {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
+            </button>
+            <div x-show="show" x-transition @click.outside="show = false" class="min-w-36 absolute block bg-white shadow-xl z-20 border-y rounded-sm end-0 top-10">
```
Comment (frjo) [inline hypha/templates/includes/login_button.html]: I think I prefer without "end-0" so the meny align with the left of the button. Personal and minor.

**Review by frjo — CHANGES_REQUESTED** [review]

File: hypha/templates/includes/login_button.html [hunk 4]
```diff
@@ -2,23 +2,68 @@
...
+                        {% trans "Activity feed" %}</a>
+                    <a href="{% url 'apply:submissions:flagged' %}" title="Goto your flagged submissions" class="pr-4 py-3 block text-black pl-3
+                                                                                                                 focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                                                 hover:bg-slate-100 hover:text-dark-blue hover:font-semibold">
+                        {% trans "My flagged" %}</a>
+                    <a href="{% url 'wagtailadmin_home' %}" title="Goto wagtail admin" class="pr-4 py-3 block text-black pl-3
+                                                                                              focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                              hover:bg-slate-100 hover:text-dark-blue hover:font-semibold">
+                        {% trans "Wagtail admin" %}</a>
```
Comment (wes-otf) [inline hypha/templates/includes/login_button.html]:
```suggestion
                        {% heroicon_mini "wrench-screwdriver" size=20 class="mr-1 inline align-text-bottom" %} {% trans "Admin" %}</a>
```

what are the thoughts on swapping this to be more inline with #3922?

**Comment by wes-otf** [comment]
Worked well in testing! I think just a few stylistic changes and it should be ready to go. Nice addition!

File: hypha/templates/includes/login_button.html (hunk 4 above)
Comment (wes-otf) [reply hypha/templates/includes/login_button.html]: I just copied the line from #3922, the tailwind may need some tweaking

File: hypha/templates/includes/login_button.html (hunk 3 above)
Comment (frjo) [reply hypha/templates/includes/login_button.html]: Does not work with short names I found out.

**Review by frjo — APPROVED** [review]

**Pull request merged**